docs/open/software.md

```diff
@@ -0,0 +1,74 @@
+* **Free Redistribution:** Open source software can be freely redistributed, without restrictions or royalties.
+* **Derived Works:** Modifications and derived works are allowed, encouraging collaborative improvement and customization.
+* **Non-Discrimination:** Open source software must be available to anyone, without discrimination against persons, groups, or fields of endeavor.
+* **License Compatibility:** Open source licenses should be compatible, allowing for the combination and integration of various open source components.
+* **Technology Neutrality:** Open source software should not be restricted to specific technologies or interfaces, promoting interoperability and platform independence.
+
+## Key Practices of Open Source Software Development
+
+* **Code sharing:** Open source software starts with publicly sharing the human-readable code that defines how a piece of software works. Source code is usually published on the internet, and often on popular code sharing platforms such as GitHub, GitLab, or GitTea.
+* **Licensing:** Granting an explicit legal license for use of source code is the foundational practice that transforms sharing into open source software. There is significant discussion amongst practitioners about the merits and priorities of different well-known public licenses
+* **Community Collaboration:** Open source projects thrive on the contributions and collaboration of a global community of developers, fostering knowledge sharing and collective problem-solving. Good practices for open source software encourage maintainers of a piece of software to include guidance on how to collaborate, including open communication channels.
+* **Open Communication:** Maintaining open communication channels (e.g., mailing lists, forums, issue trackers) for community discussions, bug reporting, and feature requests. Usually the maintainers of an open source project will establish infrastructure and patterns for open communication.
+* **Version Control:** Utilizing version control systems (e.g., Git) as a technical mechanism to manage code changes, enable collaboration, and track project history. Git is currently the predominant version control tool, and is the foundation for many code sharing platforms.
+
+## Key Practices of Research Software Engineering
+
+Research software engineering involves the development, maintenance, and management of software used in academic and scientific research. Research software engineering improves on naive research software development by embracing rigorous software engineering best practices intended to make projects more sustainable, secure, efficient, and adaptable. There are significant overlaps between research software engineering and open source software in both required practices as well as good practices. To ensure high-quality, reproducible, and sustainable research software, the following practices are commonly employed in research software enineering:
+
```

Comment:
This list is not really unique to research software engineering. These are general good practices of software engineering (with the addition of community engagement, which comes from the open source aspect). I'm not sure what I'm suggesting here, just an observation.